packages/compass-components/src/components/options-dropdown-button.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useMemo } from 'react';
+import { Link, Icon } from './leafygreen';
+import { css } from '@leafygreen-ui/emotion';
+import { spacing } from '@leafygreen-ui/tokens';
+import { uiColors } from '@leafygreen-ui/palette';
+
+const optionsButtonStyles = css({
+  backgroundColor: 'transparent',
+  border: 'none',
+  display: 'inline',
```

Comment:
Do we need this to be inline? This might cause weird line breaks for the element and doesn't look like we would want that

Reply:
Nope, removed 👍 .